androidHyperskillApp/src/main/java/org/hyperskill/app/android/step_theory/view/fragment/StepTheoryFragment.kt

```diff
@@ -118,17 +120,14 @@ class StepTheoryFragment : Fragment(R.layout.fragment_step_theory), StepCompleti
     }
 
     private fun renderSecondsToComplete(secondsToComplete: Float?) {
-        viewBinding.stepTheoryTimeToComplete.isVisible = step.secondsToComplete != null
-        if (secondsToComplete != null) {
-            val minutesToComplete = floor(secondsToComplete / 60).toInt()
-            viewBinding.stepTheoryTimeToComplete.text = resourceProvider.getString(
-                SharedResources.strings.step_theory_reading_text,
-                resourceProvider.getQuantityString(
-                    SharedResources.plurals.minutes,
-                    minutesToComplete,
-                    minutesToComplete
+        with(viewBinding.stepTheoryTimeToComplete) {
+            isVisible = secondsToComplete != null
+            if (secondsToComplete != null) {
+                text = resourceProvider.getString(
+                    SharedResources.strings.step_theory_reading_text,
+                    dateFormatter.formatHoursWithMinutesCount(secondsToComplete)
```

Comment:
Why here `formatHoursWithMinutesCount`? I suppose we only want to show `%s reading` minutes reading